hapi-fhir-storage-batch2-jobs/src/test/java/ca/uhn/fhir/batch2/jobs/imprt/BulkDataImportProviderTest.java

```diff
@@ -72,11 +75,12 @@ public void beforeEach() {
 		myProvider.setJobCoordinator(myJobCoordinator);
 	}
 
-	@Test
-	public void testStart_Success() throws IOException {
+	@ParameterizedTest
+	@ValueSource(booleans =  {true, false})
```

Comment:
minor: boolean parameters don't reveal much about the different test cases.  You could use
```suggestion
	@ValueSource(classes =  {UrlType.class, UriType.class})
```
instead and it would be obvious what the two cases are.

Reply:
Good idea, done